Interfaces/API/Job.py

```diff
@@ -1058,7 +1058,7 @@ def _handleParameterSequences(self, paramsDict, arguments):
       paramsDict['Parameters.%s' % pName]['value'] = self.parameterSeqs[pName]
       paramsDict['Parameters.%s' % pName]['type'] = 'JDL'
       if pName in self.wfArguments:
-        arguments.append(' -p %s=%%(%s)s' % (self.wfArguments[pName],
+        arguments.append(" -p %s='%%(%s)s'" % (self.wfArguments[pName],
```

Comment:
I don't think this is necessary, the list types are surrounded with curly brackets after some conversion
and later the jobAgent will replace `{` with `"{` etc.
https://github.com/DIRACGrid/DIRAC/blob/1a12b47509ebb892eb81e64686efc45b3e2a3d43/WorkloadManagementSystem/Agent/JobAgent.py#L542-L544

i.e.:
This JDL arguments
```
  Arguments = "jobDescription.xml -o LogLevel=VERBOSE  -p LogTargetPath={/ilc/prod/clic/test/431.2gev/bb/GEN/LOG/00401134/00401134_0052.tar}  -p JOB_ID=00000052  -p ProductionOutputData={/ilc/prod/clic/test/431.2gev/bb/GEN/00401134/000/bb_gen_401134_52.stdhep}  -p LogFilePath={/ilc/prod/clic/test/431.2gev/bb/GEN/00401134/LOG/000}";
```
Gets turned into this execution
```
2018-06-13 12:13:13 UTC WorkloadManagement/JobAgent DEBUG: Found standard parameter Arguments: jobDescription.xml -o LogLevel=VERBOSE  -p LogTargetPath="{/ilc/prod/clic/test/431.2gev/bb/GEN/LOG/00401134/00401134_0052.tar}"  -p JOB_ID=00000052  -p ProductionOutputData="{/ilc/prod/clic/test/431.2gev/bb/GEN/00401134/000/bb_gen_401134_52.stdhep}"  -p LogFilePath="{/ilc/prod/clic/test/431.2gev/bb/GEN/00401134/LOG/000}" 
```

Adding the quote results in this
```
2018-06-13 14:10:20 UTC WorkloadManagement/JobAgent DEBUG: Found standard parameter Arguments: jobDescription.xml -o LogLevel=VERBOSE  -p LogTargetPath='"{/ilc/prod/clic/test/431.2gev/bb/GEN/LOG/00401134/00401134_0058.tar}"'  -p JOB_ID='00000058'  -p ProductionOutputData='"{/ilc/prod/clic/test/431.2gev/bb/GEN/00401134/000/bb_gen_401134_58.stdhep}"'  -p LogFilePath='"{/ilc/prod/clic/test/431.2gev/bb/GEN/00401134/LOG/000}"' 
```

Reply:
Indeed, those quotes are already added in the JobAgent. So, is it enough to just drop this change ?

Reply:
Yes, the change in the TaskManager is enough